config/app.php

```diff
@@ -150,6 +150,7 @@
         /*
          * Package Service Providers...
          */
+         Intervention\Image\ImageServiceProvider::class,
```

Comment:
This provider would have been auto-discovered by Composer so there is no need to manually register it

Reply:
Commenting out this line works 👍